### ISSUE TITLE
aria role=table narration/navigation seems broken on Chrome

### ISSUE THREAD

**Issue by mglezer** [nvaccess/nvda#9412]
### Steps to reproduce:
1. Open Chrome
2. Browse to https://jsfiddle.net/bv4eyu5x/
3. Tab to focus on one of the two buttons in the table

### Actual behavior:
When tab-focusing on the first button (labeled "accept"), NVDA reads, `'accept  not selected  row 2  Customer ID  column 4'`. The column number is right, but the column name is not.
### Expected behavior:
When tab-focusing on the first button (labeled "accept"), NVDA reads, `'accept  not selected  row 2  fakecol1  column 4'`.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2018.4.1
#### Windows version:
Windows 10 Enterprise 1809
#### Name and version of other software in use when reproducing the issue:
Chrome Version 73.0.3683.86 (Official Build) (64-bit)
### Other questions
Standard NVDA table using `ctrl + alt + arrow` keys also does not work--not all cells are accessible.
I was unable to reproduce this problem in Firefox.

#### Does the issue still occur after restarting your PC?
Yes
#### Have you tried any other versions of NVDA?
No

**Comment by praveena-muk** [comment]
I can repro this issue in NVDA 2018.4.1  + Chrome 73.0.3683.86

**Comment by Adriani90** [comment]
This is fixed and works as expected in Chrome 81 with NVDA alpha-20220,1c4519a6. Closing as works for me.
